src/Management/test/Endpoint.Test/ActuatorRouteBuilderExtensionsTest.cs

```diff
@@ -46,10 +46,22 @@ static bool Query(Type t)
                 return t.GetInterfaces().Any(type => type.FullName == "Steeltoe.Management.IEndpoint");
             }
 
-            static bool SupportedOnPlatform(Type t)
+            static bool SupportedOnPlatform(Type type)
             {
-                return !(t.Name.StartsWith("ThreadDump") || t.Name.StartsWith("HeapDump")) || (t.Name.StartsWith("ThreadDump") && Platform.IsWindows) ||
-                    t.Name.StartsWith("HeapDump");
+                bool isThreadDump = type.Name.StartsWith("ThreadDump", StringComparison.Ordinal);
+                bool isHeapDump = type.Name.StartsWith("HeapDump", StringComparison.Ordinal);
+
+                if (!(isThreadDump || isHeapDump))
+                {
+                    return true;
+                }
+
+                if (isThreadDump && Platform.IsWindows)
```

Comment:
I think this is incorrect... ThreadDump should work on any platform since #822 (and some cleanup in #848). Also looks like the docs never got updated to indicate this changed

Reply:
Glad you're bringing this up. The result looked odd to me, so I double-checked to ensure the refactoring didn't change the outcome. What should it be instead? And which documentation are you referring to?

Reply:
I guess this line would just be `if (isThreadDump)`, but that makes the whole method look a bit weird since SupportedOnPlatform is basically always true now